next/src/utils/getLinkProps.ts

```diff
@@ -23,6 +24,7 @@ export const getLinkProps = (
     | HeaderLinkFragment
     | PageLinkFragment
     | CardLinkFragment
+    | SubnavigationItemFragment
```

Comment:
For consistency with other strapi link components, please change name from `subnavigation-item` to `subnavigation-link` (not only fragment, but the whole Strapi component.